gcloud-eclipse-tools.launch

```diff
@@ -24,7 +24,7 @@
 <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.pde.ui.workbenchClasspathProvider"/>
 <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Dosgi.requiredJavaVersion=1.7 -XX:MaxPermSize=256m -Xms256m -Xmx1024m -Doauth.client.id=${oauth_id} -Doauth.client.secret=${oauth_secret} -DenableFlexPropertySettingsPage=false"/>
 <stringAttribute key="pde.version" value="3.3"/>
-<stringAttribute key="product" value="org.eclipse.platform.ide"/>
+<stringAttribute key="product" value="org.eclipse.sdk.ide"/>
```

Comment:
what does this change mean?

Reply:
It changes the product that's launched, which primarily affects the initial perspective shown. It's why you are normally put into the _Resource_ perspective.  The `org.eclipse.sdk.ide` is what was formerly up as the _Eclipse Classic_ package.